crates/uv-settings/src/settings.rs

```diff
@@ -447,31 +447,352 @@ pub struct ResolverInstallerOptions {
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct PipOptions {
+    /// The Python interpreter into which packages should be installed.
+    ///
+    /// By default, `uv` installs into the virtual environment in the current working directory or
+    /// any parent directory. The `--python` option allows you to specify a different interpreter,
+    /// which is intended for use in continuous integration (CI) environments or other automated
+    /// workflows.
+    ///
+    /// Supported formats:
+    /// - `3.10` looks for an installed Python 3.10 using `py --list-paths` on Windows, or
+    ///   `python3.10` on Linux and macOS.
+    /// - `python3.10` or `python.exe` looks for a binary with the given name in `PATH`.
+    /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
+    #[option(
+        default = "None",
+        value_type = "str",
+        example = r#"
+            python = "3.10"
+        "#
+    )]
     pub python: Option<String>,
+    /// Install packages into the system Python.
+    ///
+    /// By default, `uv` installs into the virtual environment in the current working directory or
+    /// any parent directory. The `--system` option instructs `uv` to instead use the first Python
+    /// found in the system `PATH`.
+    ///
+    /// WARNING: `--system` is intended for use in continuous integration (CI) environments and
+    /// should be used with caution, as it can modify the system Python installation.
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = r#"
+            system = true
+        "#
+    )]
     pub system: Option<bool>,
+    /// Allow `uv` to modify an `EXTERNALLY-MANAGED` Python installation.
+    ///
+    /// WARNING: `--break-system-packages` is intended for use in continuous integration (CI)
+    /// environments, when installing into Python installations that are managed by an external
+    /// package manager, like `apt`. It should be used with caution, as such Python installations
+    /// explicitly recommend against modifications by other package managers (like `uv` or `pip`).
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = r#"
+            break-system-packages = true
+        "#
+    )]
     pub break_system_packages: Option<bool>,
+    /// Install packages into the specified directory, rather than into the virtual environment
+    /// or system Python interpreter. The packages will be installed at the top-level of the
```

Comment:
I'd say "virtual or system Python environment". No strong feelings, but want to have some consistency around language for installing with system interpreters.